src/rcdef.cpp

```diff
@@ -863,7 +865,7 @@ LOCAL void dtypes(                      // do data types
 			// process a choice type.  choicb/n format: han *choicb/n <name> {  name "text"   name "text"  ...  }
 
 			dtxnm[val] = NULL;                                   // no external name
-			dtdecl[val] = choicn ? "float" : "SI";               // ... int-->SI 2-94 to keep 16 bits
+			dtdecl[val] = choicn ? const_cast<char*>("float") : const_cast<char*>("SI");               // ... int-->SI 2-94 to keep 16 bits
```

Comment:
Is it possible to change dtdecl[ ] to const char* to avoid casting?

Reply:
I would have to define `dtdecl` right here for it to work, which will remove the purpose of the variable being global. I think it should be left how it is.